### PR TITLE
Add GUI main screen using C++

### PR DESCRIPTION
**Pull request by sadmansk**
@aneedalie just review the last commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,58 @@ Minotaur
...
+* High-speed architecture for real-time data crunching (read image recognition)  
+
+*More to come...*
+
+#### Development
+##### Setup
+For now, you only need to get the [Qt Framework](http://www.qt.io/download/). You can also get Qt Creator in order to build the code with one button and use their form editor for easy GUI design. You can open the project by specifying the folder `minotaur` and then build and run the code from there.
+
+##### Contributing
+Any help would be greatly appreciated. The entire code base is pretty big, there is a lot of code to be rewritten. If you want to help out, let [me](https://github.com/sadmansk) and we will figure something out.
```
Comment (tammy) [inline README.md]: ??? I don't see any fix...? I was talking about the grammar btw. It's supposed to be "let me know and we will figure something out"

File: minotaur/mainwindow.h [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#ifndef MAINWINDOW_H
+#define MAINWINDOW_H
+
+#include <QMainWindow>
+
+namespace Ui {
+class MainWindow;
```
Comment (tammy) [inline minotaur/mainwindow.h:7]: Not sure if this lack of indentation is on purpose. What's the reason?

**Comment by tammy** [comment]
:+1:

File: minotaur/mainwindow.h (hunk 2 above)
Comment (sadmansk) [inline minotaur/mainwindow.h:7]: That's the standard style for C++ code.

File: README.md (hunk 1 above)
Comment (sadmansk) [inline README.md]: Oops, i totally didnt see that, the hyperlinks were messed up too, ill fix those

**Pull request merged**